Galaxus.Functional.Tests/(Either)/EitherExtensionTests.cs

```diff
@@ -0,0 +1,53 @@
+using System.Collections.Generic;
+using System.Linq;
+using NUnit.Framework;
+
+namespace Galaxus.Functional.Tests
+{
+    [TestFixture]
+    public class EitherExtensionTests
+    {
+        [Test]
+        public void SelectASelectBAreEvaluatedCorrectly()
+        {
+            var eithers = new List<Either<string, int>>
+            {
+                new Either<string, int>(1),
+                new Either<string, int>("Hello"),
+                new Either<string, int>(2),
+                new Either<string, int>("World")
+            };
+
+            var aEntries = eithers.SelectA().ToList();
+            var bEntries = eithers.SelectB().ToList();
+
+            Assert.That(aEntries[0], Is.EqualTo("Hello"));
```

Comment:
Could you use a `CollectionAssert `here? That would also automatically check for the size of the list.

Reply:
This comment refers to 2 tests with the same assertion issue.